tests/robustness/traffic.go

```diff
@@ -53,6 +53,8 @@ const (
 	LeaseRevoke   TrafficRequestType = "leaseRevoke"
 	CompareAndSet TrafficRequestType = "compareAndSet"
 	Defragment    TrafficRequestType = "defragment"
+	Update        TrafficRequestType = "update"
+	Create        TrafficRequestType = "create"
```

Comment:
IT seems that the relationship between all the `TrafficRequestType` (Operation type?) is overlapped, and accordingly a little confusing. For example, 
- The Delete is reused by `traffic` and `kubernetesTraffic`, but has different meaning in different context;
- The `Update`, `Create` and `Delete` (used in KuberneteTraffic) are actualy TXN; Have you consider reusing `MultiOpTxn`? 
- `CompareAndSet` is also a TXN.

Reply:
Yea, `TrafficRequestType` is more of an identificator for `requestRandomizer` and can have different meaning for each traffic. It would be nicer if each traffic generator had specific type alias and have requestRandomizer work on generic type.